rekt.go

```diff
@@ -33,7 +33,7 @@ func evalRatelimit(author string) bool {
 		rateLimitDuration := RateLimit
 		if author == "162848980647018496" {
 			// berrely is pee pee poo poo
-			rateLimitDuration = 2 * time.Hour
+			rateLimitDuration = 0 * time.Hour
```

Comment:
maybe bump it up in the other direction instead?

Reply:
Yeah to -1

Reply:
does british "maffs" class not teach math